libraries/AP_Motors/AP_Motors6DOF.cpp

```diff
@@ -550,3 +550,29 @@ void AP_Motors6DOF::output_armed_stabilizing_vectored_6dof()
         }
     }
 }
+
+Vector3f AP_Motors6DOF::get_motor_angular_factors(int motor_number) {
```

Comment:
I think we should consider using linear factors too. In some cases a motor will not cause a torque.

Reply:
I had linear factors, they worked fine in SITL, but unfortunately I couldn't get them to work well in real life.